pom.xml

```diff
@@ -93,6 +93,33 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>flatten-maven-plugin</artifactId>
+                <version>1.7.3</version>
+                <configuration>
+                    <updatePomFile>true</updatePomFile>
+                    <flattenMode>oss</flattenMode>
```

Comment:
In Leshan we are using `ossrh` flatten mode

https://www.mojohaus.org/flatten-maven-plugin/apidocs/org/codehaus/mojo/flatten/FlattenMode.html

I understand this remove <repositories> section and so ensure we only get dependencies from maven central (add some security I guess)